client/app/types/course/assessment/submission/question/types.ts

```diff
@@ -32,7 +34,9 @@ interface TextResponseQuestionData extends TextResponseParentQuestionData {
   };
 }
 
-interface FileUploadQuestionData extends TextResponseParentQuestionData {}
+interface FileUploadQuestionData extends TextResponseParentQuestionData {
+  attachmentType: AttachmentType;
```

Comment:
there is another type of question data that relies on `TextResponseParentQuestionData `, which is `ComprehensionQuestionData`. We haven't yet fully defined the requirements for this..

I understand that at this point, we don't use this question type. But, in case we have plan in the future for implementing or refactoring this, people would be confused as to why the attachmentType and requireAttachment is also inside the struct of the Comprehension Question..

Please advise on this matter

Reply:
Good point you raise over here! I think we can keep it as what you have done for now.